### ISSUE TITLE
HUD: closing speed indicator

### ISSUE THREAD

**Issue by AxelPaccalin** [5H1N0B11/flightgear-mirage2000#87]
See screenshot proposals below by AxelPaccalin. This issue is now only about what is in the title.

**Comment by AxelPaccalin** [comment]
@5H1N0B11 You told me that the callsign was not part of the "official" reference used for the HUD, right ?

**Comment by 5H1N0B11** [comment]
I could not find any picture or something in military publication that goes in that way.

**Comment by AxelPaccalin** [comment]
But you use a picture from DCS as a ref right ?

**Comment by 5H1N0B11** [comment]
Yes. But not totally. On DCS, it is a mirage 2000C and for example the double chevron for postcombustion were'nt used at the time. But it is used now on mirage 2000-5.  Source from actual french squadron facebook who posted picture of HUD

**Comment by AxelPaccalin** [comment]
Ok, so there is a good doc for the 2000C but not much for the 2000-5

**Comment by 5H1N0B11** [comment]
exact.

**Comment by AxelPaccalin** [comment]
A first draft of the closing speed indicator sas been implemented.

Whan a tagret is under radar lock, The closing speed is shown under (if closing) the range or on top of it (if getting away).
![clos_kts](https://user-images.githubusercontent.com/47812280/88388412-c61fc680-cdb4-11ea-8a29-71266d87de08.png)

When the closing speed is getting smaller, the digits are become more precise:
![clos_kts_pre](https://user-images.githubusercontent.com/47812280/88388631-30d10200-cdb5-11ea-828b-1b38cea3d269.png)

If the target range is displayed in the metric system, the closing speed is too:
![clos_kts_m](https://user-images.githubusercontent.com/47812280/88388700-54944800-cdb5-11ea-82d3-7560c661a7d1.png)

A shot of the position opf the closing speed indicator for a tzrget getting away:
![gaway_kts_pre](https://user-images.githubusercontent.com/47812280/88388753-6c6bcc00-cdb5-11ea-8d5a-6996a80111c9.png)

The computing of the closing speed is very imprecise for now (quite "jumpy" values):  This is probably due to the fact that the physics loop and the closing-rate derivation loop are not sync.

The toogle is still not implemented.

**Comment by AxelPaccalin** [comment]
Since I'm using the new radar version, should I pull-request from the CanvasRadarPrototype or wait for it's merge before creating a new pull-request ?

**Comment by 5H1N0B11** [comment]
Indeed. Yes please :+1:

**Comment by paccalin** [comment]
Since the "locked target closing-speed indicator" is is not known as an official functionality of the m2000-5 HUD, I will add an option to toggle it.  

My question is, in which menu of the mirage should gothis toogle.
Should I create a specific menu sor the HUD (in which we could add the toogle vor various already implemented & up to come functionalities ) or add it in an already existing menu ?

**Comment by 5H1N0B11** [comment]
I was thinking about adding it to "Options"
